src/Microsoft.OData.Core/Json/IJsonReader.cs

```diff
@@ -4,6 +4,9 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+using System.IO;

```

Comment:
nit: Across this project, I see imported namespaces being nested within the namespace block